### ISSUE TITLE
How to make a floating toolbar

### ISSUE THREAD

**Issue by miqdadalik** [fraywing/textAngular#1054]
Right now, if user enter lots of content inside editor, and if they want to style paragraph in the bottom, there is no provision. Toolbar is fixed in top.

Do we have any way to overcome this issue?

**Comment by pbassut** [comment]
What does the toolbar have to do with a paragraph in the bottom? 

Enviado do meu iPhone

**Comment by miqdadalik** [comment]
If user entered lots of content inside editor, And they want to align text in the bottom, they need to scroll all the content to see the toolbar. Is there any way to overcome this?

**Comment by pbassut** [comment]
Right. There's a couple of opened issues for this. I suggest you browse around. I think someone came up with a temporary solution.

As of now, this is not supported by textAngular itself.

Enviado do meu iPhone

**Comment by AzRu** [comment]
Any luck with this @miqdadalik ? Wasn't able to find relevant issues regarding it

**Comment by miqdadalik** [comment]
For now I have made fixed toolbar. Something like attached screenshot.

And css for the same is like this.
![screen shot 2016-11-30 at 8 31 31 pm](https://cloud.githubusercontent.com/assets/8424930/20757453/5bf5fdac-b73c-11e6-96c0-12ad7235eeca.png)


```
.ta-toolbar.focussed {
    position: fixed;
    bottom: 0;
    right: 0;
    left: 0;
    z-index: 999;
    background: rgba(0,0,0,.44);
    padding: 2%;
    display: block;
}```

**Comment by JoelParke** [comment]
<p>There have been examples of this in the past...  The html looks like:</p><pre style="background-color: #ffffff;color: #000000;font-size: 9.0pt;"><span style="background-color: #efefef;">&lt;</span><span style="color: #000080;background-color: #efefef;"><b>div </b></span><span style="color: #0000ff;background-color: #efefef;"><b>text-angular-toolbar class=</b></span><span style="color: #008000;background-color: #efefef;"><b>&#34;toolbar&#34; </b></span><span style="color: #0000ff;background-color: #efefef;"><b>name=</b></span><span style="color: #008000;background-color: #efefef;"><b>&#34;toolbar&#34;</b></span><span style="background-color: #efefef;">&gt;&lt;/</span><span style="color: #000080;background-color: #efefef;"><b>div</b></span><span style="background-color: #efefef;">&gt;</span><br/></pre><pre style="background-color: #ffffff;color: #000000;font-size: 9.0pt;"><p><span style="background-color: #efefef;">&lt;</span><span style="color: #000080;background-color: #efefef;"><b>div </b></span><span style="color: #0000ff;background-color: #efefef;"><b>text-angular<br/></b></span><span style="color: #008000;background-color: #efefef;"><b>     </b></span><span style="color: #0000ff;background-color: #efefef;"><b>name=</b></span><span style="color: #008000;background-color: #efefef;"><b>&#34;&lt;your name&gt;&#34;<br/></b></span><span style="color: #008000;background-color: #efefef;"><b>     </b></span><span style="color: #0000ff;background-color: #efefef;"><b>ta-target-toolbars=</b></span><span style="color: #008000;background-color: #efefef;"><b>&#34;toolbar&#34;<br/></b></span><span style="color: #008000;background-color: #efefef;"><b>     </b></span><span style="color: #0000ff;background-color: #efefef;"><b>ng-model=</b></span><span style="color: #008000;background-color: #efefef;"><b>&#34;content.html&#34;&#10;     ....<span id="selectionBoundary_1480555564025_16265758716450507" class="rangySelectionBoundary">&#65279;</span></b></span><span style="background-color: #efefef;">&gt;</span><br/><span style="background-color: #efefef;">&lt;/</span><span style="color: #000080;background-color: #efefef;"><b>div</b></span><span style="background-color: #efefef;">&gt;</span><br/></p></pre><p>And then the toolbar can be placed where ever you want it in the html.  And you need to allow scroll bars for the &lt;div text-angular&gt;.</p><p>And https://github.com/fraywing/textAngular/wiki/Customising-The-Toolbar may help.</p>

**Comment by yatishbalaji** [comment]
```
.ta-toolbar.focussed {
  position: absolute;
  bottom: 0;
  right: 0%;
  left: 12%;
  z-index: 999;
  background: rgba(255, 254, 254, 0.90);
  padding: 2%;
  display: block;
}
```

**Comment by tyrubinstein** [comment]
****
